lib/foreman_inventory_upload/generators/slice.rb

```diff
@@ -97,13 +97,24 @@ def report_system_profile(host)
             end.join(', '))
           end
         end
-        @stream.array_field('installed_packages', :last) do
+        @stream.array_field('installed_packages') do
           first = true
           host.installed_packages.each do |package|
             @stream.raw("#{first ? '' : ', '}#{@stream.stringify_value(package.nvra)}")
             first = false
           end
         end
+        @stream.array_field('facts', :last) do
```

Comment:
facts should live under `host` node, as a sibling to [`system_profile`](https://github.com/laviro/foreman_inventory_upload/pull/87/files#diff-bc5727daebf3e0d3a95eec0e1b9f4975R55). Not inside the profile itself.